src/site/antora/modules/ROOT/pages/release-instructions-project.adoc

```diff
@@ -91,7 +91,22 @@ You can iterate on the `release/7.8.0` branch to perfect it.
 .. {site-url-staging}-7.8.0[The release staging website] is deployed using the freshly populated `release/7.8.0-site-stg-out` branch
 
 +
-If not, commit necessary fixes, push, and repeat.
+If not, commit the necessary fixes, push, and repeat.
+
+[#verify-release]
+== Verify the release
+
+Once the release is staged, the Release Manager should:
+
+. Close the _Staging Repository_ in https://repository.apache.org/[repository.apache.org] using a message that uniquely identifies the release, for example `awesome-project version 7.8.0 RC1`.
```

Comment:
This is not needed, this is automatically done by `nexus-staging-maven-plugin` – see the associated changes in ceb3e5e1a19621bc0d628760c6e82299bc1ea337.

```suggestion
```

Reply:
Thanks